docs/topics/Gateway.md

```diff
@@ -781,6 +790,29 @@ A user's presence is their current state on a guild. This event is sent when a u
 >info
 >The streaming type currently only supports Twitch. Only `https://twitch.tv/` urls will work.
 
+###### Game Timestamps
+
+| Field | Type | Description |
+|-------|------|-------------|
+| start? | int | Epoch millis of start |
+| end? | int | Epoch millis of end |
+
+###### Game Party
+
+| Field | Type | Description |
+|-------|------|-------------|
+| id? | string | The ID, relevant for joining the party |
+| size? | array of two integers | First element is the current size, second the max `[2, 5]` |
+
+###### Game Assets
+
+| Field | Type | Description |
+|-------|------|-------------|
+| large_image? | snowflake | ID for an asset of the application |
```

Comment:
images are not guaranteed to be snowflakes

Reply:
So the type should be `snowflake/string` or just `string`?

Reply:
Should I add a reference on how to format the image url using this information? What else can this be apart from snowflake?